Jenkinsfile_CNP

```diff
@@ -206,5 +206,5 @@ withPipeline(type , product, component) {
                 reportName           : "Stitching API Contract Test Report"
         ]
     }
-    syncBranchesWithMaster(['demo', 'perftest', 'ithc'])
+//    syncBranchesWithMaster(['demo', 'perftest', 'ithc'])
```

Comment:
why ?

Reply:
As mentioned in the ticket. We do not want any infrastructure changes from master to be applied to other branches till the activity is completed. We first apply the BD index on each environment separately via code and then do it on AAT & Prod. AAT & Prod is done in last via CHG.